ViewModelKit/VMKTableViewRowActionCreator.m

```diff
@@ -21,13 +21,46 @@ - (instancetype)initWithTableView:(UITableView *)tableView rowActionsType:(id<VM
     return self;
 }
 
-- (nullable NSArray<UITableViewRowAction *> *)tableViewRowActions {
+#pragma mark - UISwipeActionsConfiguration
+
+- (nullable UISwipeActionsConfiguration *)swipeActionsConfiguration {
+    NSMutableArray<UIContextualAction *> * actions = [NSMutableArray array];
+    for (VMKTableViewRowActionViewModel *rowAction in self.rowActionsType.rowActions) {
+        UIContextualAction *tableViewAction = [UIContextualAction contextualActionWithStyle: rowAction.contextualActionStyle
+                                                                                      title: rowAction.title handler:^(UIContextualAction * _Nonnull action, __kindof UIView * _Nonnull sourceView, void (^ _Nonnull completionHandler)(BOOL)) {
+            VMKViewModel *viewModel = [rowAction swipedRowAction];
+            BOOL isRowModified = NO;
+            if (viewModel) {
+                // positions were voided
+                VMKTableViewDataSource *tableViewDataSource = self.tableView.dataSource;
+                BOOL shouldOverrideSourceViewForSwipeAction = [rowAction.delegate shouldOverrideSourceViewForSwipeAction];
+                UIView * cellView = sourceView.superview.superview; //UISwipeActionPullView->_UITableViewCellSwipeContainerView
```

Comment:
Not a blocker but would be a good practice to add the class check here whatever its called 
via NSKindOfClass etc..
UISwipeActionPullView or UITableViewCellSwipeContainerView
@tovnixon